Project.toml

```
@@ -1,7 +1,7 @@
 name = "CompatHelper"
 uuid = "aa819f21-2bde-4658-8897-bab36330d9b7"
 authors = ["Dilum Aluthge", "Brown Center for Biomedical Informatics", "contributors"]
-version = "3.0.0"
+version = "3.1.0"
```

Since it's a bug fix.

```suggestion
version = "3.0.1"
```

---

I guess it's a README bug fix